### PR TITLE
feat: add import statement format rule

### PR DESCRIPTION
**Pull request by 0xRaccoon**
Signed-off-by: 0xRaccoon <raccoon@defi.sucks>

### CODE REVIEW

File: rules/import-statement-format.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+class ImportStatementFormatChecker extends BaseChecker {
+  constructor(reporter) {
+    super(reporter, ruleId, meta)
+  }
+
+  ImportDirective(node) {
+    if (!node.symbolAliases) {
+      const name = node.parent.children[node.parent.children.length - 1].name;
+      this.error(node, `Import '${node.path}' in contract ${name} should be declared as import {contract_to_import} from 'path/file_to_import.sol';`);
```
Comment (gas1cent) [inline rules/import-statement-format.js]: If possible, let's insert `node.path` insread of `'path/file_to_import.sol'`.

File: test/rules/import-statement-format.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import assert from 'assert';
+import linter from '../utils/parsers';
+const { times } = require('lodash');
+
+const config = {
+  rules: { 'import-statement-format': 'error' },
```
Comment (0xGorilla) [inline test/rules/import-statement-format.ts:6]: how do we decide whether it is warn or error?

File: test/rules/import-statement-format.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    const code = contractWithImports('import {contract_to_import} from "path/test.sol";');
+    const report = linter.processStr(code, config);
+
+    assert.equal(report.errorCount, 0);
+  });
+});
+
+function contractWithImports(code) {
+  return `
+      pragma solidity 0.4.4;
```
Comment (0xGorilla) [inline test/rules/import-statement-format.ts:30]: hehe
```suggestion
      pragma solidity 4.2.0;
```

File: test/rules/import-statement-format.ts (hunk 2 above)
Comment (0xRaccoon) [reply test/rules/import-statement-format.ts:6]: this is decided in the rule file

**Review by 0xGorilla — APPROVED** [review]

**Pull request merged**